samples/features/sql-big-data-cluster/connectivity/webhdfs-java-client/README.md

```diff
@@ -0,0 +1,64 @@
+# Building this example in Linux
+
+## Prepare dependencies jar into work dir
+
+bash ./prepare.sh
+## Building jars
+
+bash ./build.sh
+
+# Running this sample
+
+The App.java has hardcoded values in it mentioned in the sections below. Those should be pointed to the right values.
+
+Before running this application, the kinit which somes with the JRE needs to be run to get a kerberos token.
+
+`<JRE_INSTALL_ROOT>\bin\kinit.exe <user>@<REALM>`
+
+e.g. 
+
+`<JRE_INSTALL_ROOT>\bin\kinit.exe admin@ARIS.LOCAL`
+
+In Linux, you can directly using kinit in system.
+
+e.g.
+
+ kinit admin@ARIS.LOCAL
+
+# Sample login.conf. There is an existing login.conf that is being checked in as well.
+
+For more details about login.conf refer https://docs.oracle.com/javase/7/docs/technotes/guides/security/jgss/tutorials/LoginConfigFile.html
+
+```
+com.sun.security.jgss.login {
+    com.sun.security.auth.module.Krb5LoginModule required client=TRUE useTicketCache=true debug=true;
```

Comment:
In this markdown, the login.conf should show that the principal="admin" can be specified in the login.conf itself.